configure.ml

```diff
@@ -924,9 +924,9 @@ let cflags_sse2 = ["-msse2"; "-mfpmath=sse"]
 let cflags, sse2_math =
   let _, slurp =
     (* Test SSE2_MATH support <https://stackoverflow.com/a/45667927> *)
-    tryrun "cc" (["-march=native"; "-dM"; "-E"]
-                 @ cflags_sse2
-                 @ [coqtop/"kernel/byterun/coq_interp.h"] (* any file *)) in
+    tryrun "gcc" (["-march=native"; "-dM"; "-E"]
```

Comment:
Does this mean that Coq will not support clang/icc/msvc?  Or that configure will be out of sync when using those compilers?  Or was configure previously out of sync and we always use gcc?

Reply:
Weren't we already not supporting other compilers than gcc (due to the VM)?